core/src/test/java/com/graphhopper/routing/AbstractRoutingAlgorithmTester.java

```diff
@@ -592,7 +592,7 @@ public void testViaEdges_BiGraph() {
     public void testViaEdges_WithCoordinates() {
         GraphHopperStorage ghStorage = createTestStorage();
         Path p = calcPath(ghStorage, 0, 1, 2, 3);
-        assertEquals(Helper.createTList(9, 1, 2, 8), p.calcNodes());
```

Comment:
virtual nodes get a different id because of the hashmap `edge2res` in QueryGraph creates them in a different order